test/pageobjects/myaccount.page.js

```diff
@@ -37,18 +36,16 @@ class MyAccountPage extends Page {
         return $('button[type="submit"][name="login"]');
     }
 
-    async getErrorMessageText() {
-        await browser.waitUntil(async () => {
-            const errorMessageElement = this.errorMessage;
-            return await errorMessageElement.isDisplayed();
-        }, {
-            timeout: 5000,
-            timeoutMsg: 'Error message element is not displayed within 5 seconds'
-        });
-
+    getErrorMessageText() {
         const errorMessageElement = this.errorMessage;
+        errorMessageElement.waitForDisplayed({
```

Comment:
This is an async method, so `await` should be used, inherently the whole method will be `async`. Also, no need to assign value to `errorMessageElement`, you should be able to use `this.errorMessage` without another initialization.